services/create_project.py

```diff
@@ -0,0 +1,82 @@
+import requests
+import argparse
+import sys
+import click
+from typing import List
+
+sys.path.append("../")
+from models import *
+
+
+def valid_project_name(name: str, valid_initials: List[str]) -> bool:
+    if len(name) < 4:
+        return False
+    if not (name[0].lower() in valid_initials):
+        return False
+    try:
+        proj_int = int(name[1:])
+        return True
+    except:
```

Comment:
Just to be specific about what exception you're trying to catch.

```suggestion
    except ValueError:
```